models/ILSVRC2012/frontier_models.json

```diff
@@ -3,120 +3,56 @@
     "platform": "pi3",
     "frontier_models": [
       [
-        "dsb_I64x64x3CMCMCMCMCCCMC1AS",
+        "dstackedbottom",
```

Comment:
do we still want the canonical model names instead of the series names?

Reply:
or in addition to?

Reply:
I think we should keep both. I've created a story (1036) to keep track of this and will work on it on a separate PR.